### DISCUSSION TITLE
MARTES 11 de Abril: Ver videos de HASH universal.

### DISCUSSION THREAD

**Discussion by harpomaxx** [harpomaxx/algoritmos2#16] [Anuncios]
El martes seguimos con el tema de hash-tables. En esta clase vamos a ver el tema de Hash Universal.

**Comment by ExeTcod** [comment]
Hola profe buenas tardes disculpe la molestia, yo no he podido entregar el
tp trie porque estoy de viaje por trabajo, vuelvo la semana que viene.
Quería saber si me podía dar un tiempito para entregarlo porque me ha
interesado el tema y lo estoy haciendo de a poco, me falta la 2da parte y
termino.
Muchas gracias por su atención!
